sda-commons-starter-s3/src/test/java/org/sdase/commons/spring/boot/s3/config/S3HealthIndicatorIntegrationTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.s3.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doThrow;
+
+import org.junit.jupiter.api.Test;
+import org.junitpioneer.jupiter.SetSystemProperty;
+import org.sdase.commons.spring.boot.s3.S3TestApp;
+import org.sdase.commons.spring.boot.web.testing.s3.S3Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.test.web.server.LocalManagementPort;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.annotation.DirtiesContext;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.HeadBucketRequest;
+
+@SetSystemProperty(key = "management.health.s3.enabled", value = "true")
+@SpringBootTest(
+    classes = S3TestApp.class,
+    webEnvironment = WebEnvironment.RANDOM_PORT,
+    properties = {"auth.disable=true", "opa.disable=true", "management.server.port=8071"})
```

Comment:
NIT: As this is a SpringBootTest, isn't it possible to set it as property?

```suggestion
@SpringBootTest(
    classes = S3TestApp.class,
    webEnvironment = WebEnvironment.RANDOM_PORT,
    properties = {"auth.disable=true", "opa.disable=true", "management.server.port=8071", "management.health.s3.enabled=true"})
```

Reply:
Unfortunately no, because otherrwise it would ovewrite the `@SetSystemProperty` in the `checkThatS3HealthCheckIsNotEnabled` test.